### ISSUE TITLE
Accessibility: Filter panel: aria-label attribute is not allowed on the generic role

### ISSUE THREAD

**Issue by msiewierska** [shopware/shopware#13992]
### Shopware Version

6.7.3.1

### Affected area / extension

Platform(Default)

### Actual behaviour

Criteria 4.1.2 (level A) is violated: filter panel: aria-label attribute is not allowed on the generic role

<img width="1162" height="262" alt="Image" src="https://github.com/user-attachments/assets/6c4e6e92-ae97-4495-8c8f-2525a671f596" />

### Expected behaviour

_role="region"_ could be use on .filter-panel div. 

### How to reproduce

Enter listing page with filters, perform accessibility check on filters area.
